sites/website/src/docs/design-systems/high-contrast.md

```diff
@@ -0,0 +1,195 @@
+ `.withBehaviors(
+    forcedColorsStylesheetBehavior(
+        css`
+            :host {
+                background: ButtonFace;
+            }
+        `
+    )
+);
+```
+
+### System Color Keyword
+In `forced-colors` mode, the colors on the component are reduced to a limited color pallete chosen by the user. The system color keyword exposes these user chosen colors.
+
+Below are the system color keywords we use in FAST. The keywords are used as color values on style properties.
+
+
+| Keyword         | Description                                                                       |
+|-----------------|-----------------------------------------------------------------------------------|
+| `Canvas`          | Background of application content or documents.                                   |
+| `CanvasText`      | Text in application content or documents.                                         |
+| `LinkText`        | Text in non-active, non-visited links. For light backgrounds, traditionally blue. |
+| `VisitedText`     | Text in visited links. For light backgrounds, traditionally purple.               |
+| `ActiveText`      | Text in active links. For light backgrounds, traditionally red.                   |
+| `ButtonFace`      | The face background color for push buttons.                                       |
+| `ButtonText`      | Text on push buttons.                                                             |
+| `Field`           | Background of input fields.                                                       |
+| `FieldText`       | Text in input fields.                                                             |
+| `Highlight`       | Background of selected items/text.                                                |
+| `HighlightText`   | Text of selected items/text.                                                      |
+| `GrayText`        | Disabled text. (Often, but not necessarily, gray.)                                |
+
+
+FAST uses the [SystemColor](https://github.com/microsoft/fast/blob/master/packages/utilities/fast-web-utilities/src/system-colors.ts) enum when setting the color value keyword in the stylesheet.
+
+**Example**
+```css
+export const ComponentStyles = css`
+    /* ... */
+ `.withBehaviors(
+    forcedColorsStylesheetBehavior(
+        css`
+            :host {
+                background: ${SystemColors.ButtonFace};
+            }
+        `
+    )
+);
+```
+
+### Forced colors and Windows High Contrast themes
+`forced-colors` works with Windows high contrast, located in Ease of Access within Settings. There are two default themes to test high contrast, `High Contrast Black` and `High Contrast White`.
+
+![High contrast black theme](https://res.cloudinary.com/dm4izfqmy/image/upload/v1607550781/highContrast_examples/hc-document/hc-black_kkp16d.png)
+![High contrast white theme](https://res.cloudinary.com/dm4izfqmy/image/upload/v1607550781/highContrast_examples/hc-document/hc-white_fnktij.png)
```

Comment:
Need to confirm with @awentzel whether we can have these external image links.

Reply:
Yeah, I really don't want to use the external links. I tried following the document to add images here, https://github.com/microsoft/fast/blob/master/build/operations/README.md, under Azure CDN, but was not successful. @awentzel could you help? Not sure what I am missing. The use of the external link is temporary.

Reply:
The link is updated. The only way to see if the images will show up is if they are pushed to master.